packages/ERTP/src/amountMath.chainmail

```diff
@@ -25,98 +26,92 @@ struct Extent {
 }
 
 /**
- * Operations on Units.
+ * Logic for manipulating amounts.
  *
- * Units are the canonical description of tradable goods. They are manipulated
- * by mints, and represent the goods and currency carried by purses and
+ * Amounts are the canonical description of tradable goods. They are manipulated
+ * by issuers and mints, and represent the goods and currency carried by purses and
  * payments. They can be used to represent things like currency, stock, and the
  * abstract right to participate in a particular exchange.
  *
  */
-interface UnitOps (Units (Brand, Extent)) {
+interface AmountMath (Amount (Brand, Extent)) {
 
-  /** Return this unitOps's brand. */
+  /** Return the brand. */
   getBrand() -> (Brand);
 
-  /** Get the name of the extentOps used. */
-  getExtentOpsName -> (String);
+  /** Get the name of the mathHelpers used. */
+  getMathHelpersName -> (String);
 
-  /**  Make units from an extent by adding the brand. */
-  make(allegedExtent :Extent) -> (Units);
+  /**  Make an amount from an extent by adding the brand. */
+  make(allegedExtent :Extent) -> (Amount);
 
   /**
-   * Make sure these units (or extent) are valid and return them if so.
+   * Make sure this amount (or extent) is valid and return it if so.
    */
-  coerce(allegedUnitsOrExtent :Units) -> (Units);
+  coerce(allegedAmountOrExtent :Amount) -> (Amount);
 
   /** Remove the brand and return the extent. */
-  extent(units :Units) -> (Extent);
+  extent(amount :Amount) -> (Extent);
 
-  /** Return the units representing an empty amount. This is the
-   * identity element in ExtentOps.
+  /** Return the amount representing an empty amount. This is the
+   * identity element in MathHelpers.
```

Comment:
It is also the identity for the `add` method below, which is much more relevant. That is the sense it which it is an algebraic identity element.

Reply:
That's a good point. It's the identity element for the monoid `add`